test/dotnet-core-uninstall.Tests/Shared/Configs/CommandLineConfigsTests.cs

```diff
@@ -12,7 +12,7 @@
 
 namespace Microsoft.DotNet.Tools.Uninstall.Tests.Shared.Configs
 {
-    public class CommandLineConfigsTests
+    public class CommandLineConfigsTests 
```

Comment:
nit: white spaces. Usually run a git diff (extra white space will be red) before git add --all can help prevent this problem